.github/dependabot.yml

```diff
@@ -9,3 +9,5 @@ updates:
       # pin ZooKeeper dependencies to 3.5.x
       - dependency-name: "org.apache.zookeeper"
         versions: "[3.6,)"
+      # Keep commons-io at 2.6 until https://issues.apache.org/jira/browse/IO-741 is resolved
+      - dependency-name: "commons-io:commons-io"
```

Comment:
Can we create a github issue for us to track adding an integration test with symlinks so that we can validate the version bump for commons-io works in the future

Reply:
opened https://github.com/apache/druid/issues/11393 to track issue of adding integration tests.